src/ape/properties.rs

```diff
@@ -203,7 +213,15 @@ where
 	let compression_level = header_first.read_u16::<LittleEndian>()?;
 
 	// Unused
-	let _format_flags = header_first.read_u16::<LittleEndian>()?;
+	let format_flags = header_first.read_u16::<LittleEndian>()?;
+	// https://github.com/fernandotcl/monkeys-audio/blob/5fe956c7e67c13daa80518a4cc7001e9fa185297/src/MACLib/MACLib.h#L74
+	let bit_depth = if (format_flags & 0b1) == 1 {
```

Comment:
```suggestion
	let bit_depth = if (format_flags & 0x01) == 1 {
```

Reply:
Why? I think it's much more clearer if we keep it in binary format. Especially latter. It highlights the intention.

Reply:
This was just for consistency's sake, since we use hex everywhere else. It's fine though if you think it's clearer.